packages/ses/src/eval-scope.js

```diff
@@ -0,0 +1,36 @@
+import {
+  FERAL_EVAL,
+  assign,
+  create,
+  freeze,
+  defineProperties,
+} from './commons.js';
+
+export const createEvalScope = () => {
+  const evalScope = create(null);
+
+  const oneTimeEvalProperties = freeze(
+    assign(create(null), {
+      eval: freeze(
+        assign(create(null), {
```

Comment:
What is the double `assign(create(null), ...)` thing about? Please explain in a comment in the code.

Reply:
Is the part of the defense against `Object.prototype.value = something`? Given our current stance, well, I can see why we might want to at least preserve the protection right here. But `assign(create(null), ...)` seems overly complicated. Why not `{__proto__: null, ...}`?

Reply:
Or, given our general stance, I'd also be ok with removing the protection here as well, unless there's an additional reason to worry that I'm missing.

Reply:
Whatever you do here wrt the `assign(create(null), ...)`, please do it for all occurrences that serve the same purpose.

Reply:
Yeah I think `__proto__: null` may be more readable.

```js
  const oneTimeEvalProperties = freeze({
    __proto__: null,
    eval: freeze({
      __proto__: null,
      get() {
        delete evalScope.eval;
        return FERAL_EVAL;
      },
      enumerable: false,
      configurable: true,
    }),
  });
```

(and I guess same for the `setGlobalObjectSymbolUnscopables` property descriptor.

Reply:
Per #1329, we’ve elected to actually leave the original in place, no `__proto__` at all, since only a poorly vetted shim can invalidate this code. I’ve added a commit that removes the test that motivated this dance.